pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -101,7 +101,8 @@ type containerManagerImpl struct {
 	qosContainers    QOSContainersInfo
 	periodicTasks    []func()
 	// holds all the mounted cgroup subsystems
-	subsystems *cgroupSubsystems
+	subsystems *CgroupSubsystems
+	nodeInfo   *api.Node
```

Comment:
per my other comments, I think this should be an interface that lets you get at the capacity information without the entire node object.  Its fine to make this a no-op interface right now since your current PR is not actually using this information for anything yet.